packages/driver/cross-origin-testing.md

```diff
@@ -143,7 +209,7 @@ Nesting **cy.origin()** inside the callback is not currently not supported, but
 
 ### cy.intercept()
 
-We will likely add support for **cy.intercept()** within the **cy.origin()** callback in the future.
+It's not currently supported, but we will likely add support for **cy.intercept()** within the **cy.origin()** callback in the future.
```

Comment:
Should we mention the work around here? setup intercepts outside of the cy.origin block

Reply:
Mentioned and added a link to the tracking issue: https://github.com/cypress-io/cypress/pull/24508/commits/f60f9518aaffb31429cb30e56f962d19534608e1